### PR TITLE
Rename Tokens - Light Theme Prep

### PR DESCRIPTION
**Pull request by arniebradfo**
## 🗣 Description ##
I've updated the names of the tokens to make them a bit easier to read in-code and to match our Figma mockups style names.

## 💭 Motivation and context ##

The Tokens values were previously all one massive object that included all the Blueprint tokens in addition to the CoreTokens that are most used. This PR separates the tokens into 2 Token constants:
1. `CoreTokens` what should be used in 90% of cases. This now contains much fewer options with better naming
2. `AdvancedTokens` contains all the Blueprint Tokens and anything else technical

The `CoreTokens` have also  now been mapped to css `--custom-properties` to support light/dark theming! The theme values have not been fully set up yet, but they do work as a proof of concept (see screenshot). There is also a theme switch toggle that is currently commented out. It will appear in the `SettingsForm` under the ⚙️ left bar button (if you uncomment for testing).

## 🧪 Testing ##

Please take a look at the `SettingsForm` and `settings` mobx and review how the settings are applied and saved. I mostly copied how the timezone was done. I will reference specific lines in comments.

## 📷 Screenshots (Not in this PR) ##

Screen Shot 2023-01-18 at 8 23 59 AM

The colors for most of the app still need tuning. The Graph and timeline don't yet have theme values applied.

### CODE REVIEW

File: applications/client/src/store/settings.ts [hunk 1]
```diff
@@ -27,3 +39,28 @@ export class Settings extends Model({
 	}
 	momentTz = computedFn((d: MomentInput | Moment) => moment(d).tz(this.timezone));
 }
+
+// @Austin, does this belong here?
```
Comment (arniebradfo) [inline applications/client/src/store/settings.ts:43]: @GoldingAustin Please have a look at this file

File: applications/client/src/components/Forms/SettingsForm.tsx [hunk 2]
```diff
@@ -71,39 +69,15 @@ export const SettingsForm = observer<SettingsFormProps>(({ ...props }) => {
 				}}
 				label="Show Hidden Beacons, Host, and Servers"
 			/>
-			{/* <Switch
-				checked={state.theme === 'dark'}
-				onChange={state.setTheme}
-				label={'Dark theme'}
+			{/* <Switch // Uncomment to test light theme
```
Comment (arniebradfo) [inline applications/client/src/components/Forms/SettingsForm.tsx:72]: Theme toggle is commented out here. Uncomment to preview the Light Theme.

**Comment by arniebradfo** [comment]
@sang2925, @ccarpenter28, and @GoldingAustin,
Not sure why the test is failing. I didn't change anything in the UI other than colors. Could this failure have come from develop?

**Comment by sang2925** [comment]
@arniebradfo the tests are no longer failing.

**Review by GoldingAustin — APPROVED** [review]

**Pull request merged**